src/pybind/mgr/dashboard/frontend/src/app/shared/models/cephfs-directory-models.ts

```diff
@@ -0,0 +1,18 @@
+export class CephfsSnapshot {
```

Comment:
It seems as if these could be types or interfaces as they are not instantiated anywhere.

Reply:
Hi, the current typescript code is pretty outdated ATM.